moto/logs/models.py

```diff
@@ -725,6 +735,16 @@ def delete_subscription_filter(self, log_group_name, filter_name):
 
         log_group.delete_subscription_filter(filter_name)
 
+    def start_query(self, log_group_names, start_time, end_time, query_string):
+
+        for log_group_name in log_group_names:
+            if log_group_name not in self.groups:
+                raise ResourceNotFoundException()
```

Comment:
Can we get a unit test for this scenario as well?

Reply:
Done